admin-dev/themes/new-theme/js/pages/tax/display-in-cart-option-handler.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+const $ = window.$;
+
+class DisplayInCartOptionHandler {
+  constructor() {
+    DisplayInCartOptionHandler.handle();
```

Comment:
i'm not sure if it's the right thing to make static method and call it inside constructor. @matks @tomas862 wdyt?

Reply:
I agree - it should not be static in this case

Reply:
Same here: if possible, let's avoid static functions (sometimes they are needed, but not here), it makes code less reusable, less extendable and harder to unit test